tendermint/src/block.rs

```diff
@@ -51,13 +51,16 @@ where
         precommits: Option<Precommits>,
     }
 
-    let commit = TmpCommit::deserialize(deserializer)?;
-    if commit.block_id.is_none() || commit.precommits.is_none() {
-        Ok(None)
+    if let Some(commit) = <Option<TmpCommit>>::deserialize(deserializer)? {
```

Comment:
Cool, thanks! I want to double check if the tendermint rpc will return an empty commit (likely), or, if this can be treated as invalid input instead.

Reply:
An empty commit happens in the `last_commit` field in the first block.